Main.java

```diff
@@ -1,15 +1,29 @@
-/**
+import java.util.Scanner;
+
+/** 
  *
- * @author 
+ * @author Gavin Sandhar
  */
 public class Main {
 
   /**
-   * The method that is executed when you hit the run button.
+   * The method that is executed when you hit the run button
    * @param args the command line arguments
    */
   public static void main(String[] args) {
-    // TODO code application logic here
-    
+    
+    // declare a variable that will store a temperature
+   String name;
+   System.out.println("Please enter a temperature you would like to convert in degrees Celsius:");
+    
+    // Creates a Scanner used for input
+    Scanner input = new Scanner(System.in);
+     int C = input.nextInt();
+    
+    // declare and calculate the temperature
+   int temperatureF = (C*9)/5 +32;
+  
+   // Tell the user the temperature
+   System.out.println(temperatureF);

```

Comment:
Good work on the comments. Remember that a variable should always be named beginning with a lowercase letter. Also, C is the unit that appears at the end of the temperature. This doesn't make a great name. Instead, follow your lead for your other variable and name it temperatureC. This is a better description of what it is storing. Finally, try and match our output to look like the sample. This is why your code didn't pass the test cases.